plugins/poll/spec/lib/polls_updater_spec.rb

```diff
@@ -1,428 +1,117 @@
 require 'rails_helper'
 
 describe DiscoursePoll::PollsUpdater do
-  let(:user) { Fabricate(:user) }
-
-  let(:post_with_two_polls) do
```

Comment:
Some test cases have been removed here and the test cases are still relevant to me IMO.